types/test.tsx

```diff
@@ -268,7 +268,7 @@ export function testErrorHandlers() {
   render(null, {
     // Should work with React 19 types
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-expect-error
+    // @ts-ignore
```

Comment:
Since v19 types are installed by default and work fine with v19 types, the TS compiler reports that the `@ts-expect-error` directive is unused, and we should remove it, but for compatibility with the v18 types in tests, we finally decided to replace the directive with `@ts-ignore`...

Reply:
Done, thanks for review.